### PR TITLE
OCPBUGS-15443: [enterprise-4.13] Issue in file installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

### PR DESCRIPTION
**Pull request by johnwilkins**
Added a discussion of load balancing to the overview.

Fixes: OCPBUGS-15443

See https://issues.redhat.com/browse/OCPBUGS-15443 for additional details.

Preview URL: http://184.23.213.161:8080/OCPBUGS-15443/installing/installing_bare_metal_ipi/ipi-install-overview.html

For release(s): 4.14, 4.13, 4.12, 4.11, 4.10
QE Review: 

- [x] QE has approved this change. 

Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

**Comment by johnwilkins** [comment]
@cybertron Can you have a look at this? I just put a brief description of the API VIP and load balancing as part of the overview page, because there really isn't any configuration work for anyone to do. So it's really not a procedure.

File: installing/installing_bare_metal_ipi/ipi-install-overview.adoc [hunk 1]
```diff
@@ -6,23 +6,28 @@ include::_attributes/common-attributes.adoc[]
...
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_1.png[Deployment phase one]
 
 For the installation, the key elements in the previous diagram are:
 
-- **Provisioner**: A physical machine that runs the installation program and hosts the bootstrap VM that deploys the controller of a new {product-title} cluster.
+- **Provisioner**: A physical machine that runs the installation program and hosts the bootstrap VM that deploys the control plane of a new {product-title} cluster.
 - **Bootstrap VM**: A virtual machine used in the process of deploying an {product-title} cluster.
 - **Network bridges**: The bootstrap VM connects to the bare metal network and to the provisioning network, if present, via network bridges, `eno1` and `eno2`.
+- **API VIP**: An API virtual IP address (VIP) is used to provide failover of the API server across the control plane nodes. The API VIP first resides on the bootstrap VM. `systemd` manages a `keepalived` instance. A script generates the `keepalived.conf` configuration file before launching the service. The VIP moves to one of the control plane nodes after the bootstrap process has completed and the bootstrap VM stops.
```
Comment (cybertron) [inline installing/installing_bare_metal_ipi/ipi-install-overview.adoc:18]: Generally fine, but systemd isn't running keepalived. It's a static pod managed by kubelet and crio.

**Review by cybertron — COMMENTED** [review]
One small thing, but otherwise lgtm.

**Comment by cybertron** [comment]
/lgtm

Thanks!

**Review by skrthomas — APPROVED** [review]
/lgtm

**Pull request merged**